sysutils/synergy-devel/Portfile

```diff
@@ -1,55 +0,0 @@
-PortSystem 1.0
-PortGroup cmake 1.0
-
-name            synergy-devel
```

Comment:
Instead of deleting the file completely one should make a stub port with obsolete portgroup, `replaced_by synergy`. Search for `obsolete` in other ports to see examples. You only need a few lines.